packages/server-wallet/src/models/objective.ts

```diff
@@ -139,6 +144,7 @@ export class ObjectiveModel extends Model {
           data: objectiveToBeStored.data,
           createdAt: new Date(),
           progressLastMadeAt: new Date(),
+          waitingFor: objectiveToBeStored.waitingFor, // the return value is null
```

Comment:
I'm not sure I understand this comment?

Reply:
Whoops, looks like I never finished writing this comment. Probably I should explain here the choice of `null` over `undefined`.